src/app/planos/componet/planos.component.ts

```diff
@@ -8,24 +11,59 @@ import { PlanosService } from '../service/planos.service';
   styleUrls: ['./planos.component.css']
 })
 export class PlanosComponent implements OnInit {
+  roleUser: boolean = false;
+  isPremium!: Assinante;
+  ifPremium: boolean = false;
 
   constructor(
     private planosService: PlanosService,
-    private router: Router
+    private router: Router,
+    public assinanteService: AssinanteService,
+    public recommendationService: RecommendationService,
+
   ) { }
 
   ngOnInit(): void {
     this.planosService.ngOnInit();
+    this.roleIfClient();
+    this.AssinanteIfPremium();
     this.planosService.getValidation().subscribe(
-      (response) => {}, 
+      (response) => { },
       (err) => {
         if (err.status == 401) {
           this.router.navigate(['']);
-        }else if(err.status == 500){
+        } else if (err.status == 500) {
           this.router.navigate(['/erro']);
         }
       }
     );
   }
 
+  roleIfClient() {
+    this.recommendationService.getRole().subscribe(
+      (response) => {
+        console.warn('response Client', response.role);
+        if (response.role === 'Client') {
+          this.roleUser = true;
+        } else {
+          this.roleUser = false;
+        }
+      }
+    );
+  }
+
+  AssinanteIfPremium() {
+    this.assinanteService.getAssinaturaPremium().subscribe(
+      (response) => {
+        console.warn('response artista premium', response);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8399665)
```suggestion
        console.warn("response artista premium", response);
```